src/data/sparse_page_writer.h

```diff
@@ -131,8 +190,21 @@ struct SparsePageFormatReg
  *   });
  * \endcode
  */
+#define SparsePageFmt SparsePageFormat<SparsePage>
```

Comment:
The definition is out of necessity or convenience?

Reply:
The `DMLC_REGISTRY_REGISTER` macro doesn't like angle brackets.